lib/cloudevents/kafka_binding/v_1_0/decoder.ex

```diff
@@ -61,8 +87,14 @@ defmodule Cloudevents.KafkaBinding.V_1_0.Decoder do
   # ---
 
   defp content_type(headers) do
-    for({"content-type", content_type} <- headers, do: content_type)
-    |> hd()
-    |> String.downcase()
+    content_type = for({"content-type", content_type} <- headers, do: content_type)
+
+    if length(content_type) == 1 do
+      content_type
+      |> hd()
+      |> String.downcase()
+    else
+      nil
```

Comment:
I think this shouldn't return `nil` but `application/cloudevents+json` instead, cause this is what we default to (the `case` expression above reads nicer then as well).

Reply:
Ye, makes sense.